leveler/leveler.py

```diff
@@ -2854,7 +2860,13 @@ async def _write_unicode(text, init_x, y, font, unicode_font, fill):
             image = await r.content.read()
         rank_background = BytesIO(image)
         rank_avatar = BytesIO()
-        await user.avatar_url_as(format=AVATAR_FORMAT).save(rank_avatar, seek_begin=True)
+        try:
+            await user.avatar_url_as(format=AVATAR_FORMAT).save(rank_avatar, seek_begin=True)
+        except (discord.NotFound, discord.HTTPException):
+            blank_avatar_url = "https://i.imgur.com/8Pi7FBH.png"
```

Comment:
![:sonathink:](https://cdn.discordapp.com/emojis/458658772311867393.png?size=32) This doesnt looks like an [default Discord avatar](https://cdn.discordapp.com/embed/avatars/1.png).
Also perhaps, i think, better to make this link an constant variable and move it out of class.

*However, im not sure about that this is even good idea to make another webrequest in this case, as it can fail too... Perhaps, maybe better to add local copy of "default" avatar and use it in case of ~~fire~~ fail?*